proto/osmosis/concentrated-liquidity/pool-model/query.proto

```diff
@@ -44,6 +44,12 @@ service Query {
     option (google.api.http).get =
         "/osmosis/concentratedliquidity/v1beta1/positions/{address}";
   }
+  // UserPositions returns all concentrated postitions of some address.
```

Comment:
Comment is off

Reply:
https://github.com/osmosis-labs/osmosis/pull/4209/commits/14372a5abf09e4d22474e64c921c8b476bd3b282